cms/templates/cms/home_page.html

```diff
@@ -68,180 +74,169 @@
-        </div>
-      </section>
-      <section class="row current-programs-section">
-        <h3>MITx MicroMasters Programs</h3>
-        <ul class="current-programs-list">
-        {% for program, programpage in programs %}
-          <li class="col-md-4">
-            <div class="program-thumbnail">
-              {% if programpage %}
-                  <a href="{{ programpage.url }}" class="program-link">
-              {% endif %}
-                <div class="program-info">
-                  <h4 class="program-title" aria-describedby="program-{{ program.id }}-description">
-                    {% if programpage %}
-                      {{ programpage.title }}
-                    {% endif %}
-                  </h4>
-                  {% with program.course_set.count as course_count %}
-                    {% if course_count %}
-                      <div class="program-num-courses">
-                        {% blocktrans count counter=course_count %}
-                          {{ counter }} course
-                        {% plural %}
-                          {{ counter }} courses
-                        {% endblocktrans %}
-                      </div>
-                    {% endif %}
-                  {% endwith %}
-                </div>
-                {% if programpage and programpage.thumbnail_image %}
-                  {% image programpage.thumbnail_image fill-690x530 as thumbnail_image %}
-                  <img src="{{ thumbnail_image.url }}" alt=""
-                       >
-                {% else %}
-                  <img src="{% static 'images/course-thumbnail.png' %}"
-                       alt="" class="program-default-image">
-                {% endif %}
-              {% if programpage %}
-                </a>
-              {% endif %}
-            </div>
-            <div class="program-description">
-              <p class="program-description-text" id="program-{{ program.id }}-description">{{ program.description|default:"No description available for this program." }}</p>
-
-              {% if programpage %}
-                <p class="program-description-link">
-                  <a href="{{ programpage.url }}" class="program-link">
-                    More about {{ program }}
-                  </a>
-                </p>
-              {% endif %}
-            </div>
-          </li>
-        {% endfor %}
-
-        </ul>
-        <br clear="all">
-
-        <div class="more-coming-soon">
-          <p>MIT plans to bring more MicroMasters programs to motivated learners like you. Sign up for our mailing list to hear about new MicroMasters programs.</p>
-          <p>
-            <a data-toggle="modal" data-target="#subscribeAlert" class="sign-up-link">Sign Up for Mailing List</a>
-          </p>
-        </div>
-
-      </section>
-
       <section class="row reasons-section">
         <div class="centered-content">
-          <h3>Why should I enroll?</h3>
+          <h3>Why Should I Enroll?</h3>
```

Comment:
According to the invisio design, this section has some light-maroon colored rectangles in the background, don't know if they are relevant anymore but mentioning here for fyi.

Reply:
I think that's an artifact of color testing. Please ignore.